### ISSUE TITLE
Question: is there a max channel count limitation?

### ISSUE THREAD

**Issue by ruurdadema** [andrewrk/libsoundio#100]
Is there a maximum channel count limitation with libsoundio?

In the macro definitions there is a macro:

`#define 	SOUNDIO_MAX_CHANNELS   24`

Is it possible to increase this one without trouble?

I would like to use 64 channels in and 64 channels out using libsoundio

**Comment by andrewrk** [comment]
Yes. Currently there is a maximum channel count of 24. However I think we can lift this restriction in a future release. We can have the struct preallocate as part of the struct some number, maybe just 8 or even 2, and then if the number of requested channels is higher, we can allocate memory for them.
